libs/algo/odc/algo/_masking.py

```diff
@@ -427,27 +429,31 @@ def mask_cleanup(
     could be faster when using Dask, as we fuse those operations into single Dask task.
 
     :param mask: Binary image to process
-    :param filter: dict of integer - dict(closing=int, opening=int, dilation=int), order of operation if value `>0`
-        closing(optional) = remove small holes in cloud - morphological closing
-        opening = shrinks away small areas of the mask
-        dilation = adds padding to the mask
+    :param mask_filters: iterable tuples of morphological operations - ("<operation>", <radius>) - to apply on mask, where
+        operation: string, can be one of this morphological operations -
+                closing  = remove small holes in cloud - morphological closing
+                opening  = shrinks away small areas of the mask
+                dilation = adds padding to the mask
+                erosion  = Erosion shrinks bright regions and enlarges dark regions
+        radius: int
     :param name: Used when building Dask graphs
     """
 
     data = mask.data
     if dask.is_dask_collection(data):
         if name is None:
-            if filter.get("closing", 0) != 0:
-                name = f"mask_cleanup_{filter['closing']}_{filter['opening']}_{filter['dilation']}"
-            else:
-                name = f"mask_cleanup_{filter['opening']}_{filter['dilation']}"
+            name = "mask_cleanup"
+            r = []
+            for operation, radius in mask_filters:
```

Comment:
it is common to use `_` as a variable name for components you do not expect to use
```suggestion
            for _, radius in mask_filters:
```